odl/solvers/functional/functional.py

```diff
@@ -1388,7 +1371,9 @@ def __init__(self, functional, point, subgrad=None):
         >>> space = odl.uniform_discr(0, 1, 10)
         >>> l2_squared = odl.solvers.L2NormSquared(space)
         >>> point = space.one()
-        >>> Bregman_dist = odl.solvers.BregmanDistance(l2_squared, point)
+        >>> subgrad = l2_squared.gradient(point)
+        >>> Bregman_dist = odl.solvers.BregmanDistance(
```

Comment:
Even though it's a name, I'd say the rule to write variables in lower case overrules.